articles/introduction-to-evolutionary-algorithms/index.md

```diff
@@ -0,0 +1,404 @@
+### Introduction to Evolutionary Algorithms
+A computer program is built to follow rigid and explicit rules.
+This prescriptive approach has aided us well. But, it requires that the programmers know the exact steps to solve the problem and run it in this lifetime. If we swerve a bit from this strict method, we will stumble across the domain of heuristic algorithms. And in that, we find a powerful, population-based optimization algorithm, the Evolutionary algorithm (EA).
+
+An evolutionary algorithm mimics one of nature's most fascinating processes, natural selection. Given a population, only fitter members will survive and produce new generations.
+Inspired by the model of evolution by Darwin in 1859, an EA can be used to approximate solutions. This can be used even when the most optimized approach known is still not fast enough to meet real-world constraints [ Ex: [NP-Complete](https://www.ics.uci.edu/~eppstein/161/960312.html) ].
+
+### Prerequisites
+The reader must have a good understanding of the Python Programming Language.
+<br/>You can start [here](https://www.section.io/engineering-education/a-beginners-guide-to-python/)
+
```

Comment:
Is this all that is needed to understand the article? I believe a good understanding of algorithms is also necessary

Also, avoid adding <br/> in your article. You can just tap the "Enter" key on your keyboard to add space

Reply:
> Hi @Tvashta Thank you for submitting this article. Being your first article, I'm impressed. Not much revisions needed. Also, please take note of the commits I've made so that you don't make the same mistakes in your next article.
> However, I've pointed out some few sections where you need to correct. Please work on those then we can proceed for final review.

Thank you so much @Bayler for your time and help. I will correct the issues mentioned